index.js

```diff
@@ -653,7 +624,7 @@ const ContainerService = {
         userContextId: args.userContextId,
         url: object.url,
         nofocus: args.nofocus || false,
-        window: args.window || null,
+   //TOCHECK     window: args.window || null,
```

Comment:
@groovecoder we need to check the default behaviour of tab.create works as if this is passed. I suspect there are some edge cases I have not tested yet but it seems to work.